src/AppRoot.tsx

```diff
@@ -456,6 +471,37 @@ class AppRoot extends React.Component<Props, State> {
     return true
   }
 
+  public startStatusPolling = () => {
+    /* istanbul ignore else */
+    if (!this.statusPoller) {
+      this.statusPoller = IntervalPoller.start(
+        GLOBALS.CARD_POLLING_INTERVAL,
+        async () => {
+          try {
+            // Possible implementation
+            // const cardReader = await this.props.cardReader.status()
+            // const battery = await this.props.battery.status()
+            // const printer = await this.props.printer.status()
+            // this.setState({
+            //   hasCardReaderAttached: cardReader.connected,
+            //   hasChargerAttached: battery.charging,
+            //   hasLowBattery: battery.charged < 0.5,
+            //   hasPrinterAttached: printer.connected,
+            //   hasPrinterError: printer.error,
+            // })
+          } catch (error) {
+            this.stopStatusPolling() // Assume backend is unavailable.
+          }
+        }
+      )
+    }
+  }
+
+  public stopStatusPolling = () => {
+    this.statusPoller && this.statusPoller.stop()
```

Comment:
```suggestion
    this.statusPoller?.stop()
```

Reply:
Oh yeah! This was copy/pasta/edit, but should have caught it.